frontend/packages/core/src/Resolver/input.tsx

```diff
@@ -41,9 +41,10 @@ const autoComplete = async (type: string, search: string): Promise<any> => {
     return { results: [] };
   }
 
+  // We lowercase the search to make the lookup case insensitive
```

Comment:
I thought we could set a flag on the request to have this be case insensitive. It seems odd that you have to pass along a lowercase string for it to be case insensitive

Reply:
synced in standup. We will make it on the backend.